internal/pkg/agent/application/paths/common.go

```diff
@@ -14,15 +14,30 @@ import (
 	"sync"
 
 	"github.com/elastic/elastic-agent/internal/pkg/release"
+	"github.com/elastic/elastic-agent/pkg/utils"
 )
 
 const (
 	// DefaultConfigName is the default name of the configuration file.
 	DefaultConfigName = "elastic-agent.yml"
+
 	// AgentLockFileName is the name of the overall Elastic Agent file lock.
 	AgentLockFileName = "agent.lock"
-	tempSubdir        = "tmp"
-	tempSubdirPerms   = 0o770
+
+	// ControlSocketName is the control socket name.
+	ControlSocketName = "elastic-agent.sock"
+
+	// WindowsControlSocketInstalledPath is the control socket path used when installed on Windows.
+	WindowsControlSocketInstalledPath = `npipe:///elastic-agent-system`
```

Comment:
Is this equivalent to the old name `\\.\pipe\elastic-agent-system`? My Google searches haven't given me an obvious answer yet.

Reply:
Yes it is. Look at our `npipe.TransformString`, we use `npipe://` inside of `SocketURLWithFallback` which you asked me to use. So this makes it consistent.